source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -188,16 +210,33 @@ def _do_request(self):
         for pos, r in enumerate(sn_routing_matrix.rows[0].routing_response):
             if r.routing_status != response_pb2.unreached:
                 duration = self._get_duration(r, places_isochrone[pos])
-                if duration < self._max_duration_to_pt:
-                    result.update({places_isochrone[pos].uri: DurationElement(duration, r.routing_status)})
+                # if the mode is car, we need to find where to park the car :)
+                if self._mode == 'car':
+                    for sp_nearby in places_isochrone[pos].stop_points_nearby:
+                        duration_to_stop_point = self._get_manhattan_duration(
+                            sp_nearby.distance, self._speed_switcher.get('walking')
+                        )
+                        durations_sum = duration + duration_to_stop_point + CAR_PARK_DURATION
+                        if durations_sum < self._max_duration_to_pt:
+                            result[sp_nearby.uri] = DurationElement(
```

Comment:
I have understood your thoughts. 
No need to create an other output, just a new field inside DurationElement :+1: